org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/CompletionHandler.java

```diff
@@ -111,4 +115,15 @@ public boolean isCanceled() {
 		}
 		return proposals;
 	}
+
+	private String[] getFavoriteStaticMembers() {
+		PreferenceManager preferenceManager = JavaLanguageServerPlugin.getPreferencesManager();
+		if (preferenceManager != null) {
+			String serializedFavorites = preferenceManager.getPreferences().getJavaContentAssistFavoriteMembers();
```

Comment:
`preferenceManager.getPreferences().getJavaContentAssistFavoriteMembers()` should return String[] directly